api/src/main/java/jakarta/data/repository/Pageable.java

```diff
@@ -79,13 +86,22 @@ public long getPage() {
         return page;
     }
 
+    /**
+     * Return the order collection
+     *
+     * @return the order collection
+     */
+    public List<Sort> getSorts() {
+        return Collections.unmodifiableList(sorts);
```

Comment:
```suggestion
        return sorts;
```

In all cases, the list is already unmodifiable, so it doesn't make sense to wrap it again in another unmodifiable list.  Instead, just directly return the value here.